demo-client-application/app/profile/page.tsx

```diff
@@ -0,0 +1,23 @@
+'use client'
+
+import {
+  useSession
+} from '@lens-protocol/react-web'
+import { Profile } from '@lens-protocol/widgets-react'
+
+export default function ProfileComponent() {
+  const { data } = useSession();
+  if (!data) return null
+  return (
+    <div className="px-10 py-14 flex flex-col items-center">
+      {
+        data.type === "WITH_PROFILE" && data.profile && (
```

Comment:
```suggestion
        data.type === SessionType.WithProfile && data.profile && (
```

use provided enums, SessionType you can import from "@lens-protocol/react-web"